openlibrary/core/ia.py

```diff
@@ -16,7 +16,7 @@
 
 logger = logging.getLogger('openlibrary.ia')
 
-IA_BASE_URL = config.get('ia_base_url')
+IA_BASE_URL = config.get('ia_base_url', 'https://archive.org')
```

Comment:
Let's add a comment to save future folks some trouble.

```suggestion
# FIXME: We can't reference `config` in module scope like this; it will always be undefined!
IA_BASE_URL = config.get('ia_base_url', 'https://archive.org')
```